service-runner/src/main/resources/config/assets.yaml

```diff
@@ -14,6 +14,36 @@ assets:
       fee_fixed: 0
       fee_percent: 0
       max_amount: 1000000
+    sep6:
+      deposit:
+        exchange_enabled: true
+        fields:
+          email_address:
+            description: email address for transaction status updates
+            optional: true
```

Comment:
❗ SEP-6 technically supports passing KYC in deposit & withdraw requests, but passing KYC data in URLs is a security concern. Lets assume all KYC data is passed via SEP-12 prior to requests to initiate transactions, and we can remove this `fields` object here.

There is a [sentence](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0006.md#fields) is the spec that backs up this approach:

> If a field is part of the KYC/AML flow handled by SEP-12 [...], there's no need to list it in /info

Reply:
That makes sense. Shall we update the protocol to remove passing in KYC data then?

Reply:
Yes, although updating the protocol isn't time-sensitive. Can you create a PR when you have some time?